ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2

```diff
@@ -411,6 +411,11 @@ host_monitoring_cron:
   minute: "30"
   job: "ops-runner -f -s 30 -n cosp.openshift.logging.check cron-send-logging-checks"
 
+- name: "Report time to reload haproxy config" 
```

Comment:
adding the cron job as part of this PR means there is a window of time between merging this PR and building/deploying the new RPMs where this cron job will be in the failing state